src/DependencyInjection/Configuration.php

```diff
@@ -633,8 +633,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                         ->arrayNode('javascripts')
                             ->defaultValue([
-                                'bundles/sonataadmin/app.js',
-                                'bundles/sonataform/app.js',
+                                'bundles/sonataadmin/app.js?v=4.36',
```

Comment:
There will be JS modification in multiple sonataAdmin version ; I'd like to avoid adding extra maintenance burden (on a bundle which already have almost not maintainers ; I personally don't use it anymore).

Wouldn't it possible to have something dynamic in SonataAdminExtension::buildJavascripts
```
$js = $this->mergeArray(
            $config['assets']['javascripts'],
            $config['assets']['extra_javascripts'],
            $config['assets']['remove_javascripts']
        );

return array_map(
     fn (string $jsFile): string => $jsFile === 'bundles/sonataadmin/app.js' ? $jsFile.$suffix : $jsFile,
     $js,
);
```
And same for css, with an auto-computed suffix ?

Reply:
Adding the suffix dynamically, like suggested here, would definitely be better.

With the current fix people that remove the default app.js via `remove_javascripts` (own bundling system, CDN, whatever reason), would need to update their config on reach release, which just complicates things.

Reply:
Indeed, changing the current default value is basically a BC break.